vimeo/VimeoUpload
# VIM-3704: README / Consumability Work

#### Ticket
**Required for Vimeans only**
[VIM-3704](https://vimean.atlassian.net/browse/VIM-3704)

#### Ticket Summary
Prepare VimeoUpload for open sourcing (as a v0.9).

#### Implementation Summary
Build out the README. Ensure license is in place. Clarify / simplify API where possible.

#### How to Test
Critique the README. Run all upload test cases. Run both example projects and confirm uploads are working.
